book-example/src/cli/serve.md

```diff
@@ -46,3 +46,12 @@ The `--dest-dir` (`-d`) option allows you to change the output directory for the
 book. Relative paths are interpreted relative to the book's root directory. If
 not specified it will default to the value of the `build.build-dir` key in
 `book.toml`, or to `./book`.
+
+#### Specify exclude patterns
+
+The `serve` command can that list of exclude patterns to use when deciding if
+file change should trigger build or not. Patterns are read from `.gitignore`
+file in the book root directory.
```

Comment:
The first sentence seems a little awkward, I'm not sure if there's a typo somewhere? 

Maybe something like this?

```
The `serve` command will not automatically trigger a build for files listed in
the `.gitignore` file in the book root directory. The `.gitignore` file may
contain file patterns described in the [gitignore
documentation](https://git-scm.com/docs/gitignore). This can be useful for
ignoring temporary files created by some editors.
```

Reply:
I've been editing the text multiple times to get as concise and understandable version as possible. Guess it was not as understandable as I thought :-)

I've used the text you have suggested as it is way clearer. I've left the note about global `.gitignore` not being used (since that point bite my). And sorry for taking that long on such simple change.

PS: If amending commits in pull-request is not the way to go in `mdBook` project, let me know.  Unless I need to do significant changes I default to amends to not spam git history with tons of 'fixed comments' commits.